index.bs

```diff
@@ -2022,13 +2048,26 @@ To <dfn>obtain a randomized source response</dfn> given a [=randomized response
 
 <h3 algorithm id="computing-channel-capacity">Computing channel capacity</h3>
 
-To <dfn>compute the channel capacity of a source</dfn> given a [=randomized response output configuration=] |config| and a double |epsilon|:
-1. Let |pickRate| be the [=obtain a randomized source response pick rate|randomized response pick rate=] with |config| and |epsilon|.
-1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] with |config|.
-1. If |states| is 1, return 0.
-1. If |states| is greater than the user agent's [=max trigger-state cardinality=], return an error.
-1. Let |p| be |pickRate| * (|states| - 1) / |states|.
-1. Return log2(|states|) - h(|p|) - |p| * log2(|states| - 1) where h is the binary entropy function [[BIN-ENT]].
+To <dfn>compute the channel capacity of a source</dfn> given a [=randomized response output configuration=] |config|, a double |epsilon|, a possibly null positive 32-bit integer |attributionScopeLimit|, and a possibly null positive 32-bit integer |maxEventStates|:
+1. If |attributionScopeLimit| is null, set |attributionScopeLimit| to 1.
+1. If |maxEventStates| is null, set |maxEventStates| to [=default max event states=].
+1. Let |numTriggerStates| be the [=obtain a set of possible trigger states|number of possible trigger states=] with |config|.
```

Comment:
Do we still need to check this against max trigger-state cardinality?

Reply:
I think so. I don't see this checked elsewhere.

Reply:
yeah but it seems to have been deleted?